### DISCUSSION TITLE
Error: 'Oops! We encountered an unexpected error. Please try again.', 'error_code': 5027, 'error_description': 'client blocked from this operation'

### DISCUSSION THREAD

**Discussion by inmispwr** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#669] [Q&A] [answered]
Describe the bug
Error: 'Oops! We encountered an unexpected error. Please try again.', 'error_code': 5027, 'error_description': 'client blocked from this operation'
![image](https://user-images.githubusercontent.com/120057921/206288653-15a2810a-1476-402b-a653-a9fba258b5a7.png)

To Reproduce: start "run.py"

Desktop (please complete the following information):
OS: win 10
Python version [latest]

**Comment by rdavydov** [comment, marked as answer]
That's because this repo is outdated.

Try my fork https://github.com/rdavydov/Twitch-Channel-Points-Miner-v2/ as it has many fixes and improvements and is constantly evolving.

**Reply by inmispwr** [reply]
spasibo Roma!
